x-pack/plugins/endpoint/public/embeddables/resolver/types.ts

```diff
@@ -130,13 +130,57 @@ export type CameraState = {
     }
 );
 
+/**
+ * This represents all the raw data (sans statistics, metadata, etc.)
+ * about a particular subject's related events
+ */
+export interface RelatedEventDataEntry {
+  relatedEvents: Array<{
```

Comment:
I thought this would make the logic/types/etc. easier to integrate after the PR that fixes the underlying issue (which is currently under review) lands. I'm open to changing it if you feel it would make this more readable.